### DISCUSSION TITLE
Honor degradation from 4kyu 728 points to 8 kyu 5 points

### DISCUSSION THREAD

**Discussion by marcinkorbec** [codewars/codewars.com#2791] [Q&A] [answered]
Yesterday I had 728 honor rank and today when I logged in I only have 5. Did anyone like you?

**Comment by hobovsky** [comment, marked as answer]
I had a brief look at your account and it seems that some of your solutions got invalidated by a moderator, because they were recognized as copies of solutions of other users.

**Reply by marcinkorbec** [reply]
it's true I found a few solutions in google, there were less than 10 of them and I did not know that you can use them, but all the kata were taken from me, those I made myself also

**Reply by Blind4Basics** [reply]
> there were less than 10 of them 

More like 20+ of them, which is like half of the solutions of your account.
You can delete your current account and start fresh, if you want. The current one will stay in its current state otherwise.
